interface-definitions/include/haproxy/tcp-request.xml.i

```diff
@@ -0,0 +1,18 @@
+<!-- include start from haproxy/tcp-request.xml.i -->
+<node name="tcp-request">
+  <properties>
+    <help>tcp-request directive</help>
+  </properties>
+  <children>
+    <leafNode name="inspect-delay">
+      <properties>
+        <help>Set the maximum allowed time to wait for data during content inspection</help>
+        <valueHelp>
+          <format>u32:1-65535</format>
+          <description>the timeout value specified in milliseconds</description>
```

Comment:
The first symbol should be in uppercase for the description

Reply:
fixed by commit 6e93888c9993c73d885fc11c5fe41bc451eef8c1, please check the latest one